services/src/pce/src/main/java/org/openkilda/pce/model/AvailableNetwork.java

```diff
@@ -187,61 +190,75 @@ public AvailableNetwork removeSelfLoops() {
      * Since flow might be already existed and occupied some isls we should take it into account when filtering out
      * ISLs that don't have enough available bandwidth.
      * @param flowId current flow id.
+     * @param ignoreBandwidth defines whether bandwidth of links should be ignored.
+     * @param flowBandwidth required bandwidth amount that should be available on ISLs.
      */
-    public void addIslsOccupiedByFlow(String flowId) {
+    public void addIslsOccupiedByFlow(String flowId, boolean ignoreBandwidth, long flowBandwidth) {
         String query = ""
-                + "MATCH (src:switch)-[fs:flow_segment{flowid: '" + flowId + "'}]->(dst:switch) "
+                + "MATCH (src:switch)-[fs:flow_segment{flowid: $flow_id}]->(dst:switch) "
                 + "MATCH (src)-[link:isl]->(dst) "
                 + "WHERE src.state = 'active' AND dst.state = 'active' AND link.status = 'active' "
                 + "AND link.src_port = fs.src_port AND link.dst_port = fs.dst_port "
+                + "AND ($ignore_bandwidth = true "
```

Comment:
```
if(!ignoreBandwitdth) {
    query += "AND ink.available_bandwidth + fs.bandwidth >= $requested_bandwidth ";
}
```
is more correct variant.

At least remove " = true" portion, it should be `"AND ($ignore_bandwidth OR link.available_bandwidth + fs.bandwidth >= $requested_bandwidth"`.